ConsoleCalculator/Calculator.cs

```diff
@@ -3,12 +3,126 @@
 namespace ConsoleCalculator
 {
     public class Calculator
-    {
-
-        public string SendKeyPress(char key)
+    {
+        public string FirstNumber = "0";
+        public string SecondNumber = "";
+        public bool isDecimal = false;
+        public char Operator = '+';
+        
+        //Basic Arrithmetic Operations
+        private void basicOperation()
         {
-            // Add your implementation here.
-            throw new NotImplementedException();
+           
+            switch (Operator)
+            {
+                //Addition of Two Numbers
+                case '+':
+                    FirstNumber = (Double.Parse(FirstNumber) + Double.Parse(SecondNumber)).ToString();
+                    break;
+                //Substration of Two Numbers
+                case '-':
+                    FirstNumber = (Double.Parse(FirstNumber) - Double.Parse(SecondNumber)).ToString();
+                    break;
+
+                //Multipication of Two Numbers
+                case 'x':
+                case 'X':
+                    FirstNumber = (Double.Parse(FirstNumber) * Double.Parse(SecondNumber)).ToString();
+                    break;
+                //Division of Two Numbers
+                case '/':
+                    if (SecondNumber == "0") //Dividing by 0
+                        throw new DivideByZeroException();
+                    FirstNumber = (Double.Parse(FirstNumber) / Double.Parse(SecondNumber)).ToString();

```

Comment:
There is no difference in you throwing exception before operation or system throwing exception in operation. Instead you could handle and validate it during taking input.